yt_dlp/extractor/gofile.py

```diff
@@ -57,23 +59,32 @@ def _real_initialize(self):
             self._TOKEN = token.value
             return
 
-        account_data = self._download_json(
-            'https://api.gofile.io/createAccount', None, note='Getting a new guest account')
+        print('Getting a new guest account')
+        get_account_data = requests.post('https://api.gofile.io/accounts', '{}')
+        account_data = get_account_data.json()
+        # account_data = self._download_json(get_account_data, None, note='Getting a new guest account')
         self._TOKEN = account_data['data']['token']
         self._set_cookie('.gofile.io', 'accountToken', self._TOKEN)
 
     def _entries(self, file_id):
         query_params = {
-            'contentId': file_id,
-            'token': self._TOKEN,
+            # 'contentId': file_id,
+            # 'token': self._TOKEN,
             'wt': '4fd6sg89d7s6',  # From https://gofile.io/dist/js/alljs.js
         }
         password = self.get_param('videopassword')
         if password:
             query_params['password'] = hashlib.sha256(password.encode('utf-8')).hexdigest()
```

Comment:
do you know if video passwords still work the same way? the pw-protected extractor test is dead

Reply:
Sorry I didn't have a chance to try it so I don't know for sure

Reply:
I just received the password protected gofile.io URL through my friend and confirmed that it still works fine. This video may not be legal in some cases but we can try it as a test

`"https://gofile.io/d/OwrbtC" --video-password "12345"`